turtlebot3c_teleop/launch/joy.launch

```diff
@@ -1,21 +1,26 @@
 <!--
- Copyright 2020 Canonical, Ltd.
+Copyright 2023 Canonical Ltd.
 
- Licensed under the Apache License, Version 2.0 (the "License");
- you may not use this file except in compliance with the License.
- You may obtain a copy of the License at
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License version 3 as
+published by the Free Software Foundation.
 
-     http://www.apache.org/licenses/LICENSE-2.0
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
 
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
 -->
 
 <launch>
+  <arg name="teleop_config" default="$(find turtlebot3c_teleop)/config/joy_teleop/joy_teleop.yaml" />
+
   <node pkg="joy" type="joy_node" name="joystick">
     <param name="autorepeat_rate" value="10" />
   </node>
-</launch>
+
+  <rosparam file="$(arg teleop_config)" command="load" />
+  <node pkg="joy_teleop" type="joy_teleop.py" name="joy_teleop"/>
```

Comment:
Please leave those nodes separated in different launch files as they initially were. One may want to launch solely one or the other.

Reply:
Should joy_teleop.launch then include the joy.launch, or would you prefer a third launchfile?